machines/bleShare/scan/scanMachine.ts

```diff
@@ -606,7 +606,16 @@ export const scanMachine =
                 },
               },
             },
-            navigatingToHome: {},
+            navigatingToHome: {
```

Comment:
Can we rename this state according to what is performed here?
The state name says that we are navigating to home but we are disconnecting